### ISSUE TITLE
Error in XCode 15 beta 5, Stored properties cannot be marked unavailable with '@available'

### ISSUE THREAD

**Issue by RAGOpoR** [omise/omise-ios#221]
Is it possible to fix this error?

<img width="1196" alt="Screenshot 2566-07-26 at 14 38 31" src="https://github.com/omise/omise-ios/assets/875062/7f717506-2a9b-4edd-93d3-0d56183f822e">

**Comment by aashishgurung** [comment]
@RAGOpoR Thank you for bringing this up the issue. We suggest that you use the stable version of XCode because the Beta versions are prone to having bugs. If the issue persist even in XCode 15 stable version, then we will look into this. Thank you.

**Comment by Supawat-yo** [comment]
I found same problem in XCode 15 stable version.

**Comment by perthcpe23** [comment]
Found the same issue on Xcode15 stable too (through Carthage).
```
.../Carthage/Checkouts/omise-ios/OmiseSDK/CreditCardFormViewController.swift:189:6: error: stored properties cannot be marked unavailable with '@available'
    @available(*, unavailable, message: "Built in support for Card.ios was removed. You can implement it in your app and call the setCreditCardInformation(number:name:expiration:) method") // swiftlint:disable:this line_length
     ^
```

**Comment by vault087** [comment]
Thanks for your feedback, the issue was fixed in version [4.24.3](https://github.com/omise/omise-ios/releases/tag/v4.24.3)
